data/xml/2008.amta.xml

```diff
@@ -0,0 +1,543 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2008.amta">
+  <volume id="keynotes" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Proceedings of the 6th Conference of the Association for Machine Translation in the Americas: Keynote Presentations</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>Waikiki, USA</address>
+      <month>October 21-25</month>
+      <year>2008</year>
+    </meta>
+    <paper id="1">
+      <title>The Triple-Advantage Factor of <fixed-case>MT</fixed-case>: Cost, Time-to-Market, and <fixed-case>FAUT</fixed-case></title>
+      <author><first>Will</first><last>Burgett</last></author>
+      <author><first>Julie</first><last>Chang</last></author>
+      <url hash="eb9b0b20">2008.amta-keynotes.1</url>
+      <pages>1-10</pages>
+    </paper>
+    <paper id="2">
+      <title>Bringing humans into the loop: Localization with <fixed-case>MT</fixed-case> at Traslán</title>
+      <author><first>Declan</first><last>Groves</last></author>
+      <url hash="9881c9aa">2008.amta-keynotes.2</url>
+      <pages>11-22</pages>
+    </paper>
+    <paper id="3">
+      <title>Machine Translation 2008: Science meets Solution</title>
+      <author><first>Mark</first><last>Tapling</last></author>
+      <url hash="b554a117">2008.amta-keynotes.3</url>
+      <pages>23-36</pages>
+    </paper>
+  </volume>
+  <volume id="papers" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Proceedings of the 6th Conference of the Association for Machine Translation in the Americas: Technical Papers</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>Waikiki, USA</address>
+      <month>October 21-25</month>
+      <year>2008</year>
+    </meta>
+    <frontmatter>
+      <url hash="ba8967b0">2008.amta-papers.0</url>
+    </frontmatter>
+    <paper id="1">
+      <title><fixed-case>S</fixed-case>panish-to-<fixed-case>B</fixed-case>asque <fixed-case>M</fixed-case>ulti<fixed-case>E</fixed-case>ngine Machine Translation for a Restricted Domain</title>
+      <author><first>Iñaki</first><last>Alegria</last></author>
+      <author><first>Arantza</first><last>Casillas</last></author>
+      <author><first>Arantza</first><last>Diaz de Ilarraza</last></author>
+      <author><first>Jon</first><last>Igartua</last></author>
+      <author><first>Gorka</first><last>Labaka</last></author>
+      <author><first>Mikel</first><last>Lersundi</last></author>
+      <author><first>Aingeru</first><last>Mayor</last></author>
+      <author><first>Alegria</first><last>Kepa SarasolaIñaki</last></author>
```

Comment:
author list duplicated

Reply:
thanks for spotting @danielgildea, not sure what happened here.